src/config/config.js

```diff
@@ -43,22 +43,28 @@ const envVarsSchema = Joi.object({
         .default(false),
     AUTH_SERVICE_REFRESH_TOKEN_MULTIPLE_DEVICES: Joi.bool()
         .default(false),
-    AUTH_SERVICE_SEED_ADMIN_USERNAME: Joi.string()
-        .alphanum()
-        .min(3)
-        .max(30)
-        .required()
-        .default('admin')
-        .description('Admin username for seeding only'),
+    AUTH_SERVICE_SEED_ADMIN_USERNAME: Joi.alternatives([
```

Comment:
`alternatives`!

I knew `Joi` had to support multiple validations, but didn't see how in a skim of the docs.